pages/api/signup.js

```diff
@@ -1,21 +1,29 @@
 const { User } = require('../../models');
+const twilio = require('../../lib/twilio');
 
 export default async (req, res) => {
   if (req.method === 'POST') {
-    // generates random 4 character long alphanumeric string
-    req.body.verification_code = 'soft';
-
-    // insert into DB here
-    User.create(req.body)
-      .then((response) => res.json(response))
-      .catch((err) =>
-        res.status(409).json({
-          httpCode: 409,
-          message: 'User with this phone already exists',
+    if (!req.body.phone || req.body.phone.charAt(0) !== '+') {
```

Comment:
I think we can accept phone numbers in all formats. I think twilio is flexible enough to send texts to numbers like: "2132547530".

Reply:
https://www.twilio.com/docs/api/errors/60200

![image](https://user-images.githubusercontent.com/2356095/79058853-e1146380-7c27-11ea-96d8-e5b7d46ee80d.png)

Reply:
ok lgtm